backend/core/src/listings/listings.module.ts

```diff
@@ -0,0 +1,13 @@
+import { Module } from "@nestjs/common"
+import { TypeOrmModule } from "@nestjs/typeorm"
+import { ListingsService } from "./listings.service"
+import { ListingsController } from "./listings.controller"
+import { Listing } from "../entity/listing.entity"
+
+@Module({
+  imports: [TypeOrmModule.forFeature([Listing])],
```

Comment:
Maybe list the other related entities here (`unit`, `attachment`?) rather than at the root level?

Reply:
Done

Reply:
Done